package.json

```diff
@@ -39,7 +39,12 @@
     "react-select": "^5.2.2"
   },
   "peerDependencies": {
-    "@chakra-ui/react": "^1.6.0",
```

Comment:
These are all the most recent versions of these packages, meaning anyone whose using a slightly older version of chakra would be forced to install duplicate dependencies.  I did some testing on the earliest versions of these packages that would work and found that these are the earliest versions that will work properly:

```json
    "@chakra-ui/form-control": "^1.0.0",
    "@chakra-ui/icon": "^2.0.0",
    "@chakra-ui/layout": "^1.0.0",
    "@chakra-ui/menu": "^1.0.0",
    "@chakra-ui/spinner": "^1.0.0",
    "@chakra-ui/system": "^1.2.0",
```

Please update them